### PR TITLE
Fixes & improvements for a smoother deployment

### PR DESCRIPTION
**Pull request by kraft001**
Mostly changes in paths

### CODE REVIEW

File: templates/ansible/roles/deploy/tasks/main.yml.erb [hunk 1]
```diff
@@ -37,30 +37,34 @@
...
-- name: Install bundle
-  command: 'bundle install --deployment --without="development test"'
+- name: Install gems
+  command: /bin/bash -lc 'bundle install --deployment --without="development test"'
   args:
     chdir: '{{ this_release_path }}'
 
 - name: Precompile assets
-  command: bundle exec rake assets:precompile chdir={{ this_release_path }}
+  command: /bin/bach -lc 'rake assets:precompile'
```
Comment (andreychernih) [inline templates/ansible/roles/deploy/tasks/main.yml.erb]: There's a typo here, should be `/bin/bash`. Also, I think we still need `bundle exec` in front.

Comment (kraft001) [inline templates/ansible/roles/deploy/tasks/main.yml.erb]: You're right, the typo was fixed!
As for `bundle exec`, we don't need it if we use `rubygems-bundler` from `rvm`, more details [here](https://rvm.io/integration/bundler)

Comment (kraft001) [inline templates/ansible/roles/deploy/tasks/main.yml.erb]: And `bundle exec` was restored for non-rvm boxes ;)

**Comment by andreychernih** [comment]
Merging in. Thanks again!

**Pull request merged**